webdriver/tests/get_computed_role/get.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from six import text_type
+
+from webdriver.error import NoSuchAlertException
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+
+def get_computed_role(session, element):
+    return session.transport.send(
+        "GET", "session/{session_id}/element/{element_id}/computedrole".format(
+            session_id=session.session_id,
+            element_id=element))
+
+
+def test_no_browsing_context(session, closed_window):
+    response = get_computed_role(session, "id")
+    assert_error(response, "no such window")
+
+
+def test_no_user_prompt(session):
+    response = get_computed_role(session, "id")
+    assert_error(response, "no such alert")
+
+
+@pytest.mark.parametrize("html,tag,expected", [
+    ("<li role=menuitem>foo", "li", "menu"),
+    ("<input role=searchbox>", "input", "searchbox"),
+    ("<img role=presentation>", "img", "presentation")])
```

Comment:
I'm a little confused by this test. By definition, elements with role=presentation should not be included in the a11y tree. So, how is it expected that this test will pass?

Reply:
Hmm, could be a bug in the test. We could remove it here.

Reply:
@cookiecrook wdyt?

Reply:
Since this was merged a few years ago, related discussion should be raised as a new issue. This one happens to have been overcome by a later change in https://github.com/web-platform-tests/wpt/pull/39618

But the new issue @jcsteh may still be relevant against the tests in [wai-aria/role/synonym-roles.html](https://github.com/web-platform-tests/wpt/blob/master/wai-aria/role/synonym-roles.html)… See [WPT.fyi results](https://wpt.fyi/results/wai-aria/role/synonym-roles.html?label=experimental&label=master&aligned), too.